go.mod

```diff
@@ -89,28 +89,28 @@ require (
 	github.com/hashicorp/raft-autopilot v0.1.3
 	github.com/hashicorp/raft-boltdb/v2 v2.0.0-20210421194847-a7e34179d62c
 	github.com/hashicorp/raft-snapshot v1.0.3
-	github.com/hashicorp/vault-plugin-auth-alicloud v0.9.0
-	github.com/hashicorp/vault-plugin-auth-azure v0.8.0
-	github.com/hashicorp/vault-plugin-auth-centrify v0.9.0
-	github.com/hashicorp/vault-plugin-auth-cf v0.9.0
-	github.com/hashicorp/vault-plugin-auth-gcp v0.10.0
-	github.com/hashicorp/vault-plugin-auth-jwt v0.11.0
+	github.com/hashicorp/vault-plugin-auth-alicloud v0.10.0
+	github.com/hashicorp/vault-plugin-auth-azure v0.9.1
+	github.com/hashicorp/vault-plugin-auth-centrify v0.10.0
+	github.com/hashicorp/vault-plugin-auth-cf v0.10.0
+	github.com/hashicorp/vault-plugin-auth-gcp v0.11.1
+	github.com/hashicorp/vault-plugin-auth-jwt v0.11.1
 	github.com/hashicorp/vault-plugin-auth-kerberos v0.5.0
-	github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.2
-	github.com/hashicorp/vault-plugin-auth-oci v0.8.0
-	github.com/hashicorp/vault-plugin-database-couchbase v0.3.1-0.20210902192635-c3ee7c5bc378
-	github.com/hashicorp/vault-plugin-database-elasticsearch v0.8.0
-	github.com/hashicorp/vault-plugin-database-mongodbatlas v0.4.0
-	github.com/hashicorp/vault-plugin-database-snowflake v0.2.1
+	github.com/hashicorp/vault-plugin-auth-kubernetes v0.11.3
+	github.com/hashicorp/vault-plugin-auth-oci v0.9.0
+	github.com/hashicorp/vault-plugin-database-couchbase v0.5.1
+	github.com/hashicorp/vault-plugin-database-elasticsearch v0.9.1
+	github.com/hashicorp/vault-plugin-database-mongodbatlas v0.5.1
+	github.com/hashicorp/vault-plugin-database-snowflake v0.3.1
 	github.com/hashicorp/vault-plugin-mock v0.16.1
-	github.com/hashicorp/vault-plugin-secrets-ad v0.10.0
-	github.com/hashicorp/vault-plugin-secrets-alicloud v0.9.0
-	github.com/hashicorp/vault-plugin-secrets-azure v0.6.3-0.20210924190759-58a034528e35
-	github.com/hashicorp/vault-plugin-secrets-gcp v0.10.2
-	github.com/hashicorp/vault-plugin-secrets-gcpkms v0.9.0
-	github.com/hashicorp/vault-plugin-secrets-kv v0.5.7-0.20211026132900-bc1c42ddb53c
-	github.com/hashicorp/vault-plugin-secrets-mongodbatlas v0.4.0
-	github.com/hashicorp/vault-plugin-secrets-openldap v0.4.1-0.20210921171411-e86105e4986d
+	github.com/hashicorp/vault-plugin-secrets-ad v0.11.1
+	github.com/hashicorp/vault-plugin-secrets-alicloud v0.10.1
```

Comment:
Please update to [vault-plugin-secrets-alicloud-v0.10.2](https://github.com/hashicorp/vault-plugin-secrets-alicloud/releases/tag/v0.10.2)

Reply:
Fixed in https://github.com/hashicorp/vault/pull/12975/commits/4438d95143b230fc3d1f8ac510afb448b72e4dd2